bundle/jsky.app.ot/src/main/java/jsky/app/ot/editor/seq/EdIteratorFolder.java

```diff
@@ -80,12 +89,22 @@ public JPanel getWindow() {
     /** Set the data object corresponding to this editor. */
     @Override public void init() {
         final ISPObservation obs = getContextObservation();
-        // No context observation if this sequence component is in a conflict
-        // folder!
+        // No context observation if this sequence component is in a conflict folder!
         if (obs != null) {
             obs.addCompositeChangeListener(obslogUpdater);
         }
 
+        // show/hide ITC result panels depending on instrument
+        final ISPObsComponent instrument = getContextInstrument();
+        if (instrument != null) {
+            final SPComponentType type = instrument.getType();
+            _w.tabbedPane.remove(itcImagingPanel.peer());
+            _w.tabbedPane.remove(itcSpectroscopyPanel.peer());
+            if (itcImagingPanel.visibleFor(type))      _w.tabbedPane.add("ITC Imaging",      itcImagingPanel.peer());
+            if (itcSpectroscopyPanel.visibleFor(type)) _w.tabbedPane.add("ITC Spectroscopy", itcSpectroscopyPanel.peer());
+        }
```

Comment:
I definitely think this is a good place for this information and the screen-shot looks very nice.  Here in this line, if the instrument is `null` should it not remove the imaging and spectroscopy panels?  Otherwise will they be left to display whatever they were set to from the previous (if any) observation that was edited?

Also, I think it's cool to hide/show them depending on instrument and ITC availability.  I think a given observation will usually be all-imaging or all-spectroscopy though.  Maybe Andy could confirm that.  If you're just doing one or the other, perhaps the irrelevant panel shouldn't show either.  I guess you don't know that until you look at the sequence though.